resources/web/map/js/trajectory.js

```diff
@@ -1,41 +1,20 @@
-var data = {
-    type: 'Feature',
-    properties: {},
-    geometry: {
-        type: 'LineString',
-        coordinates: []
-    }
-};
-
 const style = {
     "version": 8,
     "sources": {
         "osm": {
             "type": "raster",
-            "tiles": ["https://a.tile.openstreetmap.org/{z}/{x}/{y}.png"],
+            // "tiles": ["https://a.tile.openstreetmap.org/{z}/{x}/{y}.png"],
+            "tiles": ["https://api.mapbox.com/v4/mapbox.satellite/{z}/{x}/{y}.png?access_token=pk.eyJ1Ijoic3dpZnQtYWRyaWFuIiwiYSI6ImNsZTN1MW82bDA2OGgzdXFvOWFuZTJlMnEifQ.9nR8m0C-B_ISNR4r4cMExw"],
```

Comment:
this API key is free account (think it should be safe to leave in source(?) or maybe bake this into build but might be weird for setting up locally)
 
think satellite view looks pretty good, lots of cool tiles we can choose from with maplibre-gl + free BUT it wont be 'out of the box' with the two buttons like googlemaps, (would probably need to find a plugin or add buttons to switch between)